--- wmtheme-1.3.3/Makefile.orig	2004-06-13 13:38:44.000000000 +0200
+++ wmtheme-1.3.3/Makefile	2004-06-13 13:42:39.000000000 +0200
@@ -46,7 +46,7 @@
   Makefile stubwm.pl make-wmtheme apply-gtk2-theme.c wminstallfonts \
   wmtheme.spec.in make-rpm lsm.in
 
-all: apply-gtk2-theme manpage
+all: apply-gtk2-theme 
 	PREFIX=$(prefix) $(PERL) ./make-wmtheme
 
 apply-gtk2-theme: apply-gtk2-theme.c
@@ -57,13 +57,13 @@
 	for SUPT in $(SUPPORTFILES) ; do \
 	  $(INSTALL) -m 644 $$SUPT $(DESTDIR)$(LIBDIR); \
 	done;
-	$(INSTALL) -m 644 wmtheme.1.gz $(DESTDIR)$(MANDIR)
+	$(INSTALL) -m 644 wmtheme.1 $(DESTDIR)$(MANDIR)
 	for INVOCATION in $(INVOCATIONS); do \
 	  if [ ! -e $(DESTDIR)$(BINDIR)/$$INVOCATION ] ; then \
 	    ln -s wmtheme $(DESTDIR)$(BINDIR)/$$INVOCATION; \
 	  fi; \
-	  if [ ! -e $(DESTDIR)$(MANDIR)/$$INVOCATION.1.gz ] ; then \
-	    ln -s wmtheme.1.gz $(DESTDIR)$(MANDIR)/$$INVOCATION.1.gz; \
+	  if [ ! -e $(DESTDIR)$(MANDIR)/$$INVOCATION.1 ] ; then \
+	    echo ".so wmtheme.1" > $(DESTDIR)$(MANDIR)/$$INVOCATION.1; \
 	  fi; \
 	done;
 	$(INSTALL) -m 755 wminstallfonts $(DESTDIR)$(BINDIR)
